### DISCUSSION TITLE
How to hide contributions on stat card?

### DISCUSSION THREAD

**Discussion by toharifqi** [anuraghazra/github-readme-stats#1210] [Q&A] [answered]
Hi, I have dificulty on hiding the contribution value on the stat card. I can hide the stars by adding &hide=stars. But unfortunately I do not know the variabel name of contributions. Thank you.

**Comment by anuraghazra** [comment, marked as answer]
Hi @toharifqi the variable name is `contribs`

https://github-readme-stats.vercel.app/api?username=anuraghazra&hide=contribs

**Reply by toharifqi** [reply]
I got it, thank your for the quick help

**Comment by nico-i** [comment]
Please add this to the README.

**Reply by rickstaa** [reply]
@nico-i I checked, and it looks like it is already found under the section https://github.com/anuraghazra/github-readme-stats#hiding-individual-stats. Is this what you had in mind?

**Reply by nico-i** [reply]
@rickstaa oh my bad, I must have missed that, sorry. 

But would it be possible to link to the [mentioned section](https://github.com/anuraghazra/github-readme-stats#hiding-individual-stats) in the _[Stats Card Exclusive Options](https://github.com/anuraghazra/github-readme-stats#stats-card-exclusive-options)_ section? That was at least where I would have expected the input values to be, namely aside of the corresponding query paramater. 😁

**Reply by rickstaa** [reply]
@nico-i Your request looks reasonable. I agree that the documentation on this feature can be improved. If you create a feature request or pull request, I will take a look when I have the time.

**Reply by nico-i** [reply]
@rickstaa I created a new PR [here ](https://github.com/anuraghazra/github-readme-stats/pull/1766) and a new issue about the discussed changes to be made [here](https://github.com/anuraghazra/github-readme-stats/issues/1767). Cheers 🍻
